### DISCUSSION TITLE
How can I use classname instead of style with edges ?

### DISCUSSION THREAD

**Discussion by AFinne-Specks** [xyflow/xyflow#3081] [Q&A] [answered]
Hello,
Am i forced to use the style attribut of the edges ? 
I would like to use a classname attribut to use edges with tailwindcss.

Thank you.

**Comment by bcakmakoglu** [comment, marked as answer]
You *can* use a className attribute for edges, but if you're looking to style the stroke of an edge, you'll have to target that element, for example

```ts
// passing `className` to an edge
{ id: "e1-3", source: "1", target: "3", className: "my-edge-class" }
```

```css
.my-edge-class path {
  stroke: blue;
}
```

[Here's a sandbox.](https://codesandbox.io/s/beautiful-sammet-8ih3eg?file=/src/Flow.tsx:725-796)

**Reply by AFinne-Specks** [reply]
Thx for answering.

How can i apply this style to edges being dragged ? I could only use style but it duplicates the code.

Here is how my ReactFlow component is :
`    <ReactFlow
      nodes={nodes}
      edges={edges}
      onNodesChange={onNodesChange}
      onEdgesChange={onEdgesChange}
      onConnect={onConnect}
      nodeTypes={nodeTypes}
      edgeTypes={edgeTypes}
      connectionLineStyle={{ stroke: '#D83388', strokeWidth: 2 }}
      fitView
    >`

**Reply by bcakmakoglu** [reply]
Alas, you can't pass a className to the *ConnectionLine* component (unless you use a custom connection line, then you can use whatever you want) but you can just use the existing class name to add some css to it.

```css
.react-flow__connection-path {
  stroke: blue;
}
```

**Reply by AFinne-Specks** [reply]
Thanks !
